src/status.ts

```diff
@@ -24,7 +24,7 @@ abstract class Button {
   private _tooltip: string|null = null;
   private _icon: string|null = null;
 
-  constructor(protected readonly config: ConfigurationReader, private readonly _priority: number) {
+  constructor(protected readonly config: ConfigurationReader, protected readonly _priority: number) {
```

Comment:
It seems like _priority should be renamed to priority if it's no longer private.